attune/io/Topas4.py

```diff
@@ -0,0 +1,79 @@
+from .. import Instrument
+from .. import Tune
+from .. import Arrangement
+from typing import Dict, Union
+import json
+import numpy as np
+
+
+def fromTopas4(Topasfile):
```

Comment:
capitalization of both the method name and argument are fairly non standard here...

I would expect `from_topas4` and `topas_file`

I would perhaps look to accept a _folder_ and/or _zip file_ for reasons laid out belown.